draft-ietf-quic-tls.md

```diff
@@ -895,7 +895,28 @@ keeping track of missing packet numbers.
 # Packet Protection {#packet-protection}
 
 As with TLS over TCP, QUIC protects packets with keys derived from the TLS
-handshake, using the AEAD algorithm negotiated by TLS.
+handshake, using the AEAD algorithm {{!AEAD}} negotiated by TLS.
+
+QUIC packets have varying protections depending on their type:
+
+* Version Negotiation packets have no cryptographic protection.
+
+* Retry packets use AEAD_AES_128_GCM to provide protection against accidental
+  modification or insertion by off-path adversaries; see
+  {{retry-integrity}}.
+
+* Initial packets use AEAD_AES_128_GCM with keys derived from the Destination
+  Connection ID field of the client Initial packet; see {{initial-secrets}}.
```

Comment:
```suggestion
  Connection ID field of the first Initial packet sent by the client; see
  {{initial-secrets}}.
```

Reply:
I'm surprised this doesn't mention the properties provided, since the other two bullets do.  The Initial protection provides the same protections that the Retry tag does, plus protection against version-unaware middleboxes.

Reply:
I put that in the referenced section.  I wasn't sure which way to go.